### PR TITLE
feat(virtual-scroll): optimation changes

### PR DESCRIPTION
**Pull request by necoxrr**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

File: packages/renderless/src/virtual-scroll/index.ts [hunk 1]
```diff
@@ -108,6 +108,7 @@ export const handleScroll = ({ props, state, virtualScroll, nextTick, items, ...
       totalSize = calculateTotalSize({ state, props })
     }
     state.totalSize = totalSize
+    console.log('高度',state.totalSize);
```
Comment (gimmyhehe) [inline packages/renderless/src/virtual-scroll/index.ts]: 源码中，不宜出现调试的打印语句~

File: packages/renderless/src/virtual-scroll/index.ts [hunk 2]
```diff
@@ -118,7 +119,7 @@ export const handleScroll = ({ props, state, virtualScroll, nextTick, items, ...
     })
   }
 }
-
+// 更改
```
Comment (gimmyhehe) [inline packages/renderless/src/virtual-scroll/index.ts]: 此处注释看起来很笼统模糊，麻烦完善一下注释，或者删除无用注释

File: packages/renderless/src/virtual-scroll/index.ts (hunk 1 above)
Comment (necoxrr) [reply packages/renderless/src/virtual-scroll/index.ts]: 好的

File: packages/renderless/src/virtual-scroll/index.ts (hunk 2 above)
Comment (necoxrr) [reply packages/renderless/src/virtual-scroll/index.ts]: 好的

**Comment by necoxrr** [comment]
此分支用于优化，先不合并。

File: examples/sites/demos/pc/app/tree/virtual-drag-node.vue [hunk 3]
```diff
@@ -12,21 +12,23 @@
...
-      @node-drag-enter="dragEnter"
-      @node-drag-over="dragOver"
-      @node-drag-leave="dragLeave"
-      @node-drag-end="dragEnd"
-      default-expand-all
-    ></tiny-tree>
+    <div class="virtual-drag-node">
+      <tiny-tree
+        :data="data"
+        show-virtual-scroll
```
Comment (ErKeLost) [inline examples/sites/demos/pc/app/tree/virtual-drag-node.vue]: 这块属性就叫 `virutal-scroll` 吧。虚拟滚动是功能 show 更像是 展示和 隐藏某些东西

**Review by ErKeLost — APPROVED** [review]

File: examples/sites/demos/pc/app/tree/virtual-drag-node.vue (hunk 3 above)
Comment (necoxrr) [reply examples/sites/demos/pc/app/tree/virtual-drag-node.vue]: 已经更改合适属性。

**Review by kagol — APPROVED** [review]

**Pull request merged**